test/clj/rems/browser_test_util.clj

```diff
@@ -36,6 +39,11 @@
     (.mkdirs)
     (delete-files)))
 
+(def accessibility-report-dir
+  (doto (io/file "browsertest-accessibility-report")
+    (.mkdirs)
+    (delete-files)))
```

Comment:
Oh, I hadn't noticed we have load-time side-effects in this file. I'd rather we didn't use those, especially since they're not necessary. Please use a function or a delay.

Reply:
This is standard practice in the file, you can see it in action in the other dirs too. It has been there since the beginning. Is it worthwhile to change?

Reply:
I think it should be fixed. Otherwise e.g. `(reload)` can nuke your browser test failures. I think the directories should only be cleared when you start a new test run. A follow-up PR is fine too, or I can do it if you don't feel like it.

Reply:
I can do it into this PR.

Reply:
thanks